ts-obs/cmd/pgx.go

```diff
@@ -29,22 +49,29 @@ func OpenConnectionToDB(namespace, name, user, dbname string, remote int) (*pgxp
 		return nil, errors.New("user not found")
 	}
 
-	podName, err := KubeGetPodName(namespace, map[string]string{"release": name, "role": "master"})
-	if err != nil {
-		return nil, err
-	}
+	if strings.HasSuffix(host, ".svc.cluster.local") {
```

Comment:
is this the best way to determine this? Can we instead see if the db pod exists and port-forward if it does (not port-forwarding if it doesn't)? @blagojts do you have any ideas here?

Reply:
there are probably better ways to do this, but i can't think of any realistic instance where it would fail. checking the pod is possible, but i dont know if the extra complication is necessary

Reply:
For services in the same k8s cluster you can just specify the service name `ts-obs` or even `ts-obs.namespace` and it will still be resolved by the DNS service, the svc.cluster.local is not necessary. So this would fail in those cases